pandas/io/clipboard/__init__.py

```diff
@@ -124,6 +124,12 @@ def paste_osx_pbcopy():
 
 
 def init_osx_pyobjc_clipboard():
+    try:
+        import AppKit
+        import Foundation  # check if pyobjc is installed
+    except ImportError:
+        return init_osx_pbcopy_clipboard()
```

Comment:
thanks for looking into this!

not sure about returning `init_osx_pbcopy_clipboard` within `init_osx_pyobjc_clipboard`

I think pylint might not be correct here, e.g. if we have
```python
def foo():
    print(re)

def bar():
    global re
    try:
        import re
    except ImportError:
        pass
    else:
        foo()

bar()
```
then it executes just fine

Shall just keep as-is and turn off the warning?

Reply:
Thanks, @MarcoGorelli for the example. Sure thing, let’s keep this warning turn off.